frontend/actions/siteActions.js

```diff
@@ -19,6 +19,14 @@ const alertError = (error) => {
   alertActions.httpError(error.message);
 };
 
+const onUserRemoveFromSite = (site) => {
+  if (site) {
+    dispatchUserRemovedFromSiteAction(site);
+  }
+
+  alertActions.alertSuccess('User successfully removed.');
```

Comment:
Well, I guess this would be cool if it changed the message based on if it was `me`. I think you could just pass it along from `removeUserFromSite`. Maybe in a separate PR?

Reply:
Yeah agreed, I will put it in a new PR.